controller/controller.go

```diff
@@ -0,0 +1,17 @@
+package controller
+
+import (
+	"net/http"
+
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/csvdata"
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/graph"
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/model"
+	"github.com/gorilla/mux"
```

Comment:
I recommend to separate the import libraries like

import (
// standar libraries

// our libraries 

// third party libraries)

Just skipping one empty row from each section